### DISCUSSION TITLE
Estimating DBH / Basal Area from Airborne Lidar Data using lidR

### DISCUSSION THREAD

**Discussion by vaasuSS** [r-lidar/lidR#593] [Q&A] [answered]
Hi. Is there any workflow / tool with which Basal Area / DBH can be estimated using lidR? Thanks.
Reference:
[Basal Area](https://en.wikipedia.org/wiki/Basal_area) - Basal area is the cross-sectional area of trees at breast height. It is a common way to describe stand density
[DBH](https://en.wikipedia.org/wiki/Diameter_at_breast_height) - Diameter at breast height

**Comment by Jean-Romain** [comment, marked as answer]
Yes by using an area based approach and statistical modelling with `pixel_metrics` or using an individual tree segmentation approach and deriving metrics per trees related to allometric equation using `segment_trees` and `tree_metrics`. You can also read [the book](https://r-lidar.github.io/lidRbook/) for more details. However there is no built-in models.

**Reply by vaasuSS** [reply]
Thanks very much Jean. This is really helpful.
